programs/vscode.json

```diff
@@ -5,6 +5,11 @@
             "movable": false,
             "help": "Currently unsupported.\n\n_Relevant issue:_ https://github.com/microsoft/vscode/issues/3884 \n"
         },
+        {
```

Comment:
Thought that there is no need in additional file for VS Code Insiders, so moved it into `vscode.json`. 

Commit 009610a70647f6aa4ffa266227e1159eb8fa6b16 can be reverted to use `vscode-insiders.json`.